src/main/java/ca/corefacility/bioinformatics/irida/service/impl/ProjectServiceImpl.java

```diff
@@ -502,6 +502,26 @@ public void removeSamplesFromProject(Project project, Iterable<Sample> samples)
 				.build();
 	}
 
+	/**
+	 * {@inheritDoc}
+	 */
+	@Override
+	@Transactional(readOnly = true)
+	@PreAuthorize("hasRole('ROLE_USER')")
+	public List<Project> getProjectsForUserUnique(User user) {
```

Comment:
You might want to talk to Katherine about these, as they might be useful for her work with the user > projects tab on her account refresh.

Reply:
Will do!